src/NBitcoin/RPC/RestClient.cs

```diff
@@ -247,9 +219,9 @@ private WebRequest BuildHttpRequest(string resource, RestResponseFormat format,
 
             HttpWebRequest request = WebRequest.CreateHttp(uriBuilder.Uri);
             request.Method = "GET";
-#if !NETCORE
+            #if !NETCORE
```

Comment:
Prepocessor directives are usually written with no margin on the left.
But it doesn't matter, I plan to remove those soon.